tests/mvp-demo/mvp-demo.go

```diff
@@ -246,6 +254,35 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 			Expect(snapshot.Name).ToNot(BeEmpty())
 		})
 
+		It("Integration Test PipelineRun is created", func() {
+			Eventually(func() bool {
+				testPipelinerun, err = f.AsKubeAdmin.IntegrationController.GetIntegrationPipelineRun(integrationTestScenario.Name, snapshot.Name, userNamespace)
+				if err != nil {
+					GinkgoWriter.Printf("failed to get Integration test PipelineRun for a snapshot '%s' in '%s' namespace: %+v\n", snapshot.Name, userNamespace, err)
+					return false
+				}
+				return testPipelinerun.HasStarted()
+			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue())
+			Expect(testPipelinerun.Spec.PipelineRef.Bundle).To(ContainSubstring(integrationTestScenario.Spec.Bundle))
+			Expect(testPipelinerun.Labels["appstudio.openshift.io/snapshot"]).To(ContainSubstring(snapshot.Name))
+			Expect(testPipelinerun.Labels["test.appstudio.openshift.io/scenario"]).To(ContainSubstring(integrationTestScenario.Name))
+		})
+
+		It("Integration Test PipelineRun should eventually succeed", func() {
+			Eventually(func() bool {
+				testPipelinerun, err = f.AsKubeAdmin.IntegrationController.GetIntegrationPipelineRun(integrationTestScenario.Name,  snapshot.Name, userNamespace)
+				if err != nil {
+					GinkgoWriter.Printf("failed to get Integration test PipelineRun for a snapshot '%s' in '%s' namespace: %+v\n", snapshot.Name, userNamespace, err)
+					return false
+				}
+				return testPipelinerun.IsDone()
```

Comment:
I think this check is not sufficient for validating that the pipelinerun succeeded. if `.IsDone()` returns `true`, it means that pipelinerun has finished, but it could have failed. IMO this additional check needs to be added.
https://github.com/redhat-appstudio/e2e-tests/blob/ff61a4f0bbfb8740b0954d892522ac856b26e028/tests/mvp-demo/mvp-demo.go#L457

or perhaps we could use this method here?
https://github.com/redhat-appstudio/e2e-tests/blob/ff61a4f0bbfb8740b0954d892522ac856b26e028/pkg/utils/integration/controller.go#L343

Reply:
@Pavel, can you please help to review it again? Thx.